modules/learn/pages/clusters-and-availability/xdcr-conflict-logging-feature.adoc

```diff
@@ -0,0 +1,541 @@
+
+| `isDeleted`
+| True or false for - is the source/target document deleted?
+
+| `scope` and `collection`
+| Name of the scope and collection where the conflict source/target documents exist.
+
+| `revSeqno`
+| Source/target document’s revision sequence number.
+
+| `expiry`
+| Source/target document’s expiry time.
+
+| `flags`
+| Source/target document’s flag.
+
+| `xattrs`
+| System xattributes.
+
+| `_vv`
+| Document version vector.
+HLV metadata.
+|===
+
+[#conflict-logger]
+=== Logger
+
+The Logger in XDCR records conflict events detected during a replication.
+It operates as an asynchronous component designed to minimize the impact on replication performance and system resources.
+
+Conflict logging is attempted on the best-effort basis.
+If any error occurs, due to resource issues or a delay in writing to the conflict log collection, XDCR may pause or stop the logging process, report the error type for the pipeline, and continue with replication so that there is no High Availability issue.
+
+By default, the Conflict Logger is tuned to handle a low percentage of conflicts.
+
+`xdcr_clog_status` is a Prometheus stat, which indicates the status of the Conflict Logger.
+The status can be one of the following:
+
+* Hibernated: The logger is temporarily disabled due to persistent errors or resource constraints.
++
+NOTE: Hibernation prevents the degradation of the replication performance. Logging is re-enabled after a set interval, or after the errors are manually resolved and the logger is manually restarted.
+
+* Running: The logger is actively processing conflict events.
+
+* DNE: The logger does not exist.
+
+[#configure-conflictlogging-settings]
+== Configuring the conflictLogging Setting
+
+You can configure a xref:learn:clusters-and-availability/xdcr-enable-crossclusterversioning.adoc#conflict-collection[conflict collection] by using the `conflictLogging` key. 
+
+The `conflictLogging` key is an advanced XDCR setting that enables or disables conflict logging and specifies where conflict records must be stored.
+It allows you to define a default conflict collection and, optionally, detailed logging rules (through `loggingRules`) for different scopes and collections.
+By configuring `conflictLogging`, you control whether conflicts are logged, and if so, in which bucket and collection the conflict records are saved.
+For more information, see the examples and formats below.
+
+To log conflicts, when creating an XDCR replication use the following settings, which accepts a key and a corresponding JSON object as the value.
+
+* xref:learn:clusters-and-availability/xdcr-enable-crossclusterversioning.adoc#conflictlogging-key[`conflictLogging`] is the key.
+* xref:learn:clusters-and-availability/xdcr-enable-crossclusterversioning.adoc#logging-rules[`loggingRules`] is a JSON object as the value.
+
+The conflict logging settings are a part  of the xref:rest-api:rest-xdcr-adv-settings.adoc[Manage Advanced Settings] in XDCR replication.
```

Comment:
You can also update an existing replication's conflict logging settings by updating the `conflictLogging` option, which is part of the xref:rest-api:rest-xdcr-adv-settings.adoc[Manage Advanced Settings] in XDCR replication.